raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -63,53 +71,73 @@ contract NettingChannelContract {
         return data.addressAndBalance();
     }
 
+    /// @notice Get the amount one partner has sent/transferred to the other partner.
+    /// @param participant The address of one partner.
+    /// @return The amount one partner has sent to the other.
     function transferredAmount(address participant) constant returns (uint) {
         return data.transferredAmount(participant);
     }
 
+    /// @notice Close the channel. Can only be called by a participant in the channel.
+    /// @param theirs_encoded The state of the partner's transfers.
+    /// @param ours_encoded The state of the caller's transfers.
     function close(bytes theirs_encoded, bytes ours_encoded) {
         data.close(msg.sender, theirs_encoded, ours_encoded);
         ChannelClosed(msg.sender, data.closed);
     }
 
+    /// @notice Dispute the state after closing, called by a counterparty (the
+    ///         participant who did not close the channel).
+    /// @param theirs_encoded The transfer the counterparty believes is the valid
+    ///                       state of the first participant.
     function updateTransfer(bytes theirs_encoded) {
         data.updateTransfer(msg.sender, theirs_encoded);
         TransferUpdated(msg.sender, block.number);
     }
 
+    /// @notice Unlock a locked transfer.
     function unlock(bytes locked_encoded, bytes merkle_proof, bytes32 secret) {
         data.unlock(msg.sender, locked_encoded, merkle_proof, secret);
         ChannelSecretRevealed(secret);
     }
 
+    /// @notice Settle the transfers and balances of the channel and pay out to
+    ///         each participant. Must be called by a participating address.
```

Comment:
Correction here: `settle()` can be called by anyone around the world. But it has to be called only after `close()` and only after `settle_timeout` amount of blocks have passed since the channel was closed with the `close()` function.